ui/app/adapters/totp-key.js

```diff
@@ -23,65 +23,43 @@ export default ApplicationAdapter.extend({
       response.data.id = name;
       return response;
     });
-  },
-
-  createRecord() {
-    return this.createOrUpdate(...arguments);
-  },
-
-  updateRecord() {
-    return this.createOrUpdate(...arguments, 'update');
-  },
+  }
 
   deleteRecord(store, type, snapshot) {
     const { id } = snapshot;
     return this.ajax(this.urlForKey(snapshot.record.backend, id), 'DELETE');
-  },
-
-  pathForType() {
-    return 'keys';
-  },
+  }
 
   urlForKey(backend, id) {
-    let url = `${this.buildURL()}/${encodePath(backend)}/${this.pathForType()}/`;
+    let url = `${this.buildURL()}/${encodePath(backend)}/keys/`;
```

Comment:
I just noticed this ends in a slash, I don't think the base URL should end in one and instead this should happen in the 
```suggestion
    let url = `${this.buildURL()}/${encodePath(backend)}/keys`;
```

```js
if (!isEmpty(id)) {
 url = `${url}/${encodePath(id)}`;
}